kuka_kr5_support/urdf/kr5_arc_macro.xacro

```diff
@@ -0,0 +1,279 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="kuka_kr5_arc" params="prefix">
+    <!-- LINKS -->
+    <!-- base link -->
+    <link name="${prefix}base_link">
+      <inertial>
+        <origin xyz="-0.017519394031846 8.11394768450848E-07 0.0975924044854325"
+                rpy="0 0 0" />
+        <mass value="7.58585707666653" />
+        <inertia
+            ixx="0.0441803207364147"
+            ixy="5.37433593375545E-07"
+            ixz="0.004045564920652"
+            iyy="0.0584309411563095"
+            iyz="-4.8990117845347E-07"
+            izz="0.0514875586171667" />
+      </inertial>
+      <visual>
+        <origin rpy="0 0 0" xyz="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr5_support/meshes/kr5_arc/visual/base_link.dae" />
+        </geometry>
+      </visual>
+      <collision>
+        <origin rpy="0 0 0" xyz="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr5_support/meshes/kr5_arc/collision/base_link.stl" scale="0.001 0.001 0.001" />
+        </geometry>
+      </collision>
+    </link>
+    <!-- link 1 (A1) -->
+    <link name="${prefix}link_1">
+      <inertial>
```

Comment:
all the elements below this one seem to be at the same indentation level. Should they be one indentation level deeper? They appear to be children of `inertial` (and `base_link` does have them indented that way).